src/payment/payment-method-action-creator.ts

```diff
@@ -23,7 +26,7 @@ export default class PaymentMethodActionCreator {
                         sessionHash: response.headers['x-session-hash'],
                     };
 
-                    observer.next(createAction(PaymentMethodActionType.LoadPaymentMethodsSucceeded, response.body, meta));
+                    observer.next(createAction(PaymentMethodActionType.LoadPaymentMethodsSucceeded, this._filterApplePay(response.body), meta));
```

Comment:
If we were to get more conditions to filter the payment methods, should we add lines like below one at a time
```
observer.next(createAction(PaymentMethodActionType.LoadPaymentMethodsSucceeded, this._filterSOMETHINGELSE(response.body), meta));
observer.next(createAction(PaymentMethodActionType.LoadPaymentMethodsSucceeded, this._filterSOMETHINGELSE(response.body), meta));
```
Or use one function to act as a filter?

Reply:
This is the question I came up with while discussing the amazon pay response, so I am thinking we can move with this and possibly change the method and implementation based on the conditions when we look at the amazon ticket?